README.md

```diff
@@ -18,9 +18,8 @@ Full installation instructions are available in [INSTALL.md](INSTALL.md).
 General Prerequisites:
 
 * Ubuntu/Debian
-* Perl (at least version 5.10.1)
-* PostgreSQL (at least version 9.1)
-* Memcached
```

Comment:
I saw no particular reason to list memcached here, but not Redis or node. Since we’ll forget updating this anyway, e.g. at the time memcached will be dropped, better only mention the core points here.